My sql window functions (#3077)
section-engineering-education/engineering-education: content/authors/elphaze-sedah/index.md

> ---
> title: Elphaze Sedah
> type: authors
> github: https://github.com/elphazesedah
> ---
> Sedah is a student at JKUAT .He loves database administration and also likes to program. He is a singer and also plays piano.

Reviewer: ```suggestion
---
title: Elphaze Sedah
type: engineering-education/author
github: https://github.com/elphazesedah
---
Sedah is a student at JKUAT. He loves database administration and programming. He is a singer and plays piano.
```
